planemo/options.py

```diff
@@ -1377,7 +1377,7 @@ def shed_fail_fast_option():
 
 def lint_xsd_option():
     return planemo_option(
-        "--xsd/--no_xsd", is_flag=True, default=True, help=("Include tool XSD validation in linting process.")
+        "--xsd/--no_xsd", is_flag=True, default=False, help=("Include tool XSD validation in linting process.")
```

Comment:
This would trigger [planemo's built in xsd check](https://github.com/galaxyproject/planemo/blob/fcabbd6507a5d41e2ff9f1ed474278ac479fecac/planemo/lint.py#L89) which is now covered by https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/tool_util/linters/xsd.py (which runs by default).

Reply:
Is there any reason to keep the option (and the corresponding code) then ?

Reply:
My rationale was to keep the `xmllint` based [xsd check](https://github.com/galaxyproject/planemo/blob/cd7a7a1558abac40ce71c3ff27ec8258fe33abca/planemo/xml/validation.py#L76C32-L76C48).

With the (admittedly strange) command line  `--skip XSD --xsd` it should be possible to disable the galaxy-tool-util xsd linter and to enable planemo's xsd linter.

Maybe we should at least drop the `lxml` based `xsd` check and improve the help of the parameter?

Reply:
> With the (admittedly strange) command line --skip XSD --xsd it should be possible to disable the galaxy-tool-util xsd linter and to enable planemo's xsd linter.

why would you ever do that ? can we just stick with one ?